### DISCUSSION TITLE
Farewell to @oxy 😢

### DISCUSSION THREAD

**Discussion by jsjoeio** [coder/code-server#3933] [General]
We want to give a big thanks to @oxy as we bid him farewell! Unfortunately, due to visa requirements, Friday was his last day at Coder and on the code-server team. He is going back to school in the fall to finish his Computer Science degree at Miami University. Even though he's only been here since December 2020, he has had a huge impact on code-server and the community. Some of his most important work includes:
- rewriting our CI from scratch to be more stable
- reducing our build times by 50-60% (best case when we hit cache both times, which is most PRs)
- tackling VS Code updates, including updating one patch in the same week it was released
- investigating the switch to open-vsx
- migrating to cross-compiling for arm64 and also adding armv7 support for Raspberry Pis
- and many more

We're super grateful to have had him on the team with us and we hope he'll stay in touch in case he ever wants to come back to Coder after he graduates.

**Comment by sr229** [comment]
It's a sad day but I'm rooting for him, especially as a friend who watched them grow slowly during their duration on this project.

Bon voyage, @oxy! Hope you come back soon.
